pwd_guess.py

```diff
@@ -1642,7 +1675,7 @@ def pwd_complies(self, pwd):
             count += 1
         if self.has_group(pwd, self.lowercase):
             count += 1
-        if self.all_from_group(pwd, self.non_symbols):
+        if self.has_group(pwd, self.symbols):
```

Comment:
This line should count any character that is in the character bag, but not a lowercase, uppercase, or digit as a symbol. Instead, this change will only count symbols as those that are in the pre-determined list of symbols.